CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/inc
                     ${CMAKE_CURRENT_SOURCE_DIR}/lib_joystick
                     ${CMAKE_CURRENT_SOURCE_DIR}/lib_utils
                     ${CMAKE_CURRENT_SOURCE_DIR}/lib_openvslam/openvslam/src
+                    ${CMAKE_CURRENT_SOURCE_DIR}/lib_openvslam/external_dependencies/socket.io-client-cpp/lib/asio/asio/include
```

Comment:
I've specifically mentioned in the instructions that libasio is to be installed using apt get, which would put the required header in the system directories and in the default path. It's only if this is not done that you need to add this line.

Reply:
I see, I must've missed that, OK .. will update the PR